### DISCUSSION TITLE
Does Pester support "Test Owner" Property?

### DISCUSSION THREAD

**Discussion by hankyi95** [pester/Pester#2103] [Q&A] [answered]
I want to indicate the Test Owner for each Pester Test, Is there a recommended way to achieve that?

**Comment by fflaten** [comment, marked as answer]
I'd probably use tags for now, ex `It 'some test' -Tags 'Owner-fflaten' { }`.

There's a feature request to support metadata (key/value-tags) in #1469 which would be cleaner, but not something being worked on currently.

**Reply by nohwnd** [reply]
That, or you can add a testcases hashtable with any data you want, and you will have it in the output object as Data.

`It 'some test' -ForEach @{ Owner = 'fflaten'} { }`

Or maybe better use some kind of convention to distinguish real test data from you meta-data. Like prefixing with `_`.
